reporters/fail.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+var PluginError = require('gulp-util').PluginError;
+var through = require('through2');
+
+module.exports = function (opts) {
+	opts = opts || {};
+
+	// paths to files that failed JSCS
+	var fails = false;
+
+	// files that need to be passed downstream on flush
+	var buffer = opts.buffer !== false ? [] : false;
```

Comment:
I would prefer we drop it. We can always add it back later if there's people complaining.